dev/release/verify-release-candidate.sh

```diff
@@ -982,8 +982,8 @@ test_linux_wheels() {
     local pyver=${python/m}
     for platform in ${platform_tags}; do
       show_header "Testing Python ${pyver} wheel for platform ${platform}"
-      ENV=wheel-${pyver}-${platform} PYTHON_VERSION=${pyver} maybe_setup_conda || exit 1
-      ENV=wheel-${pyver}-${platform} PYTHON_VERSION=${pyver} maybe_setup_virtualenv || continue
+      VENV_ENV=wheel-${pyver}-${platform} PYTHON_VERSION=${pyver} maybe_setup_conda || exit 1
```

Comment:
Should this be CONDA_ENV here and on L1016 below? I was having trouble verifying wheels until I changed this

Reply:
It should, putting up a PR. Thanks for catching that!

Reply:
Here is the fix https://github.com/apache/arrow/pull/13061

Reply:
Oh, sorry...